origin.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Andre Lau Portfolio</title>
+    <link rel="stylesheet" href="./main.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <h3>Hector Andre Lau</h3>
+            <a href=""><img src="./images/Union.svg" alt="Menu"></a>
+        </nav>
+    </header>    
+        
+        <h1>Hey there, I'm Hector Andre Lau</h1>
+        <h2>I'm a software developer with experience in finance and logistic processes.</h2>
+        <p>I can build a website meeting the expected requirements, pluss I can add my actual knowledge of how the process need to work on certain areas like logistic, account, finance and billing and all of this focus to the good experience and satisfaction ot the customer.</p>
```

Comment:
- [ ] [OPTiONAL] Please, consider to add these `h1`, `h2` and `p` into `section` tag, which is coming exactly after, because this mentioned tags are outside of a main `section` tag - by doing it you will consider the HTML semantic structure: 
<img width="630" alt="Screen Shot 2022-02-10 at 18 32 51" src="https://user-images.githubusercontent.com/49064106/153429099-3a131900-4e44-4401-8042-377e4449e77e.png">